snuba/clickhouse/formatter/expression.py

```diff
@@ -53,7 +58,8 @@ def _alias(self, formatted_exp: str, alias: Optional[str]) -> str:
             assert ret is not None
             return ret
         else:
-            self._parsing_context.add_alias(alias)
+            if self._include_parsing_context:
```

Comment:
actually re-thinking this, I might want to still add the alias to the parsing context, but I just want to skip the above check `self._parsing_context.is_alias_present(alias):`